Dockerfile.local

```diff
@@ -1,4 +1,4 @@
-FROM python:3.12-alpine as builder
+FROM python:alpine as builder
```

Comment:
Is good to have the version well defined here, for the ensure that the pipeline will not break if a new unsupported Python version is released.